### PR TITLE
Add Open-CE

### PR DESCRIPTION
**Pull request by jqyin**

### CODE REVIEW

File: software/analytics/ibm-wml-ce.rst [hunk 1]
```diff
@@ -123,9 +134,9 @@ For more information on ``jsrun`` please see:
 Setting up Custom Environments
 ==============================
 
-The IBM-WML-CE conda environment is read-only. Therefore, users
+The IBM-WML-CE and Open-CE conda environment are read-only. Therefore, users
```
Comment (verolero86) [inline software/analytics/ibm-wml-ce.rst:137]: "conda environment" -> "conda environments"

File: software/analytics/ibm-wml-ce.rst [hunk 2]
```diff
@@ -163,12 +174,12 @@ Performance Profiling
 
 There are several tools that can be used to profile the performance of a
 deep learning job. Below are links to several tools that are available
-as part of the ibm-wml-ce module.
+as part of the ibm-wml-ce and open-ce module.
```
Comment (verolero86) [inline software/analytics/ibm-wml-ce.rst:177]: "and open-ce module" -> "and open-ce modules"

File: software/analytics/ibm-wml-ce.rst [hunk 3]
```diff
@@ -163,12 +174,12 @@ Performance Profiling
...
 
 NVIDIA Profiling Tools
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The ibm-wml-ce module contains the nvprof profiling tool. It can be used to
+The ibm-wml-ce and open-ce module contains the nvprof profiling tool. It can be used to
```
Comment (verolero86) [inline software/analytics/ibm-wml-ce.rst:182]: "module contains" -> "modules contain"

**Review by verolero86 — CHANGES_REQUESTED** [review]
Hi @jqyin ! 
Thank you for updating the docs to include `open-ce`. I added a couple minor edits. 
Best,
Verónica

File: software/analytics/ibm-wml-ce.rst (hunk 3 above)
Comment (jqyin) [reply software/analytics/ibm-wml-ce.rst]: Thanks @verolero86, I have fixed all typos.

**Review by verolero86 — APPROVED** [review]

File: software/analytics/ibm-wml-ce.rst (hunk 3 above)
Comment (verolero86) [reply software/analytics/ibm-wml-ce.rst]: Thank you @jqyin!

**Pull request merged**